CODE/Sources/PennyBOT/Handlers/MessageHandler.swift

```diff
@@ -46,33 +46,30 @@ struct MessageHandler {
             )
             do {
                 let response = try await self.coinService.postCoin(with: coinRequest)
-                if response.starts(with: "ERROR-") {
```

Comment:
A thought I've had, but maybe we could send the errors to the bot-lab channel so we don't always have to dive into the logs unless the bot crashes. I have no idea if that's possible though

Reply:
Personally I've kind of liked looking at the logs in AWS, it's convenient to me and you can expand JSONs and look at them properly.

For logging these to a channel in Discord, I personally use a Discord logger in my other project which uses Web hook to send data to a Discord channel.
I've been meaning to make it publicly available with proper configurations, but im not sure when i'll get to it.

We can add something like that if we really want to, just using normal create-message or a Web hook.